src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php

```diff
@@ -91,92 +84,143 @@ public function read(array $domains, array $locales): TranslatorBag
     {
         $domains = $domains ?: ['*'];
         $translatorBag = new TranslatorBag();
+        $responses = [];
 
         foreach ($locales as $locale) {
             foreach ($domains as $domain) {
-                $response = $this->client->request('GET', sprintf('export/locale/%s.xlf?filter=%s&status=translated', $locale, $domain));
+                $responses[] = [
+                    'response' => $this->client->request('GET', sprintf('export/locale/%s.xlf?%s',
+                        rawurlencode($locale),
+                        http_build_query([
```

Comment:
should use the `query` option instead, isn't it?
if not, `http_build_query` is missing some arguments (we don't want the separator to be configured by php.ini)